src/applications/vaos/containers/DateTimeSelectPage.jsx

```diff
@@ -63,7 +65,17 @@ const uiSchema = {
 export class DateTimeSelectPage extends React.Component {
   componentDidMount() {
     focusElement('h1.vads-u-font-size--h2');
-    this.props.openSelectAppointmentPage(pageKey, uiSchema, initialSchema);
+    const { preferredDate } = this.props;
+    this.props.openFormPage(pageKey, uiSchema, initialSchema);
+    this.props.getAppointmentSlots(
```

Comment:
Why did you decide to break this into two action creators?

Reply:
The original `openSelectAppointmentPage()` was basically a combination of the getAppointmentSlots() and `openFormPage()`.  Since we are going to be reusing the fetch, I separated that out into `getAppointmentSlots()`.

I guess I could have left `openSelectAppointmentPage()` and dispatched `getAppointmentSlots()` from within that action, but it wasn't doing anything differently than `openFormPage()`.